### PR TITLE
Proposal: Remove Autocapture functionality from Paths

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

https://github.com/PostHog/posthog/issues/5665#issuecomment-916791179 

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by macobo** [comment]
Love it! :)

**Review by paolodamico — COMMENTED** [review]
This makes sense to me, but I don't have enough context to review this part of the codebase, think it's worth waiting for @EDsCODE's comments.

File: posthog/queries/paths.py [hunk 1]
```diff
@@ -65,11 +65,7 @@ def calculate_paths(self, filter: PathFilter, team: Team):
         sessions = (
             Event.objects.add_person_id(team.pk)
             .filter(team=team, **(event_filter), **date_query)
-            .filter(
-                ~Q(event__in=["$autocapture", "$pageview", "$identify", "$pageleave", "$screen"])
-                if event is None
-                else Q()
-            )
+            .filter(~Q(event__in=["$pageview", "$identify", "$pageleave", "$screen"]) if event is None else Q())
```
Comment (neilkakkar) [inline posthog/queries/paths.py:68]: Wouldn't we want to keep `"$autocapture"` in here, since it's a Negation query? i.e. Event not in `[this list]`.

Comment (EDsCODE) [reply posthog/queries/paths.py:68]: yeah, misread

**Pull request merged**